packages/core/src/components/toast/toast.md

```diff
@@ -20,29 +13,29 @@ have more than one toast onscreen at a time.
 The `Toast` and `Toaster` components are available in the __@blueprintjs/core__ package.
 Make sure to review the [general usage docs for JS components](#blueprint.usage).
 
-The `Toaster` component provides the static `create` method that returns a new `Toaster` instance, rendered into an
-element attached to `<body>`. (You can also specify the element to render into if desired.) A `Toaster` instance
-has a collection of methods to show and hide toasts in its given container.
+@### Toast
 
-Your application can contain several `Toaster` instances and easily share them across the codebase as modules.
+`Toast`s have a built-in timeout of five seconds. Users can also dismiss them manually by clicking the &times; button.
+Hovering the cursor over a toast prevents it from disappearing. When the cursor leaves the toast, the toast's timeout restarts.
+Similarly, focusing the toast (for example, by hitting the `tab` key) halts the timeout, and blurring restarts the timeout.
 
-```tsx
-// toaster.ts
-import { Position, Toaster } from "@blueprintjs/core";
+You can add one additional action button to a toast. You might use this to undo the user's action, for example.
 
-export const OurToaster = Toaster.create({
-    className: "my-toaster",
-    position: Position.BOTTOM_RIGHT,
-});
-```
+You can also apply the same visual intent styles to `Toast`s that you can to [`Button`s](#core/components/button.css-api).
 
-```tsx
-// application.ts
-import { OurToaster } from "./toaster";
+@interface IToastProps
 
-const key = OurToaster.show({ message: "Toasted!" });
-OurToaster.update(key, { message: "Still toasted!" });
-```
+@### Toaster
+
+The `Toaster` React component is a stateful container for a single list of toasts. Internally, it
+uses [`Overlay`](#core/components/overlay) to manage children and transitions. It can be vertically
+aligned along the top or bottom edge of its container (new toasts will slide in from that edge) and
+horizontally aligned along the left edge, center, or right edge of its container.
+
+`Toaster` can be used in one of two ways:
+
+1. `Toaster.create(props)` static method returns a new `IToaster` instance. Use the instance method `toaster.show()` to manipulate this instance. __(recommended)__
+1. Render a `<Toaster>` element and either use the `ref` prop to access its instance methods, or pass rendered `<Toast>`s as `children`.
```

Comment:
This feels like 3 options.

1. `Toaster.create(...)`
2. `<Toaster ref={...} />`
3. `<Toaster><Toast ... /></Toaster>` 

they can all be mixed & matched anyway, so might as well list all 3.

Reply:
2 and 3 can be mixed, no way to `provide` children to 1.

but yes there are three, so i'll update

Reply:
well you can combine option 1 with 2 & 3, you'd just get multiple Toaster instances